### PR TITLE
Public release notes

### PR DESCRIPTION
**Pull request by i-skrypnyk**
## Description
Added release notes in md format
Can be seen here:
https://github.com/HorizenOfficial/Sidechains-SDK/blob/is/release_notes/doc/index.md

## Jira Ticket
[SDK-1393](https://horizenlabs.atlassian.net/browse/SDK-1393)

## Changes
Added release notes in md format

## Checks
- [x] Project Builds
- [x] Project passes tests and checks
- [x] Updated documentation accordingly
- [x] Breaking changes have been correctly tagged and notified
 
## Additional information

[SDK-1393]: https://horizenlabs.atlassian.net/browse/SDK-1393?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: doc/release/eon_1.0.0.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Release version - EON 1.0.0
+
+---
```
Comment (paolocappelletti) [inline doc/release/eon_1.0.0.md:3]: let's add an "Update from previous version" section with any specific instruction for the update. For this version:
- remove peers folder?
- update config file if any new parameter is different from the default

**Review by paolocappelletti — CHANGES_REQUESTED** [review]
Please also add a link to doc/index.md in the root readme

File: doc/release/eon_1.0.0.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Release version - EON 1.0.0
```
Comment (paolocappelletti) [inline doc/release/eon_1.0.0.md:1]: let's remove any reference to EON, here we are in the sdk
this release note  is for 0.8.0

File: doc/index.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Horizen Sidechain SDK Release Notes
+
+## [EON 1.0.0](/doc/release/eon_1.0.0.md) (2023-09-06)
```
Comment (paolocappelletti) [inline doc/index.md:3]: let's remove any reference to EON, here we are in the sdk
this release note is for 0.8.0

**Review by paolocappelletti — CHANGES_REQUESTED** [review]

**Review by victHorizen — CHANGES_REQUESTED** [review]

**Review by paolocappelletti — APPROVED** [review]

**Pull request merged**